src/Compilers/CSharp/Portable/Emitter/Model/ParameterSymbolAdapter.cs

```diff
@@ -29,11 +29,11 @@ bool Cci.IParameterTypeInformation.IsByReference
             }
         }
 
-        bool Cci.IParameterTypeInformation.HasByRefBeforeCustomModifiers
+        ushort Cci.IParameterTypeInformation.CountOfCustomModifiersPrecedingByRef
```

Comment:
Is there a limit on the number of custom modifiers that can appear which caused us to choose `ushort` here?